twill-yarn/src/main/java/org/apache/twill/internal/yarn/AbstractYarnAMClient.java

```diff
@@ -50,12 +51,11 @@
   private static final Logger LOG = LoggerFactory.getLogger(AbstractYarnAMClient.class);
 
   // Map from a unique ID to inflight requests
-  private final Multimap<String, T> containerRequests;
-
-  // List of requests pending to send through allocate call
-  private final List<T> requests;
+  private final Multimap<String, T> inflightRequests;
+  // Map from a unique ID to pending requests. It is for recording
```

Comment:
Oh. It is for recording the container requests that has yet to be sent to RM. Will update the comment.